frontend/src/screens/learner/card.tsx

```diff
@@ -1,44 +1,50 @@
 import { cx, React, useState } from '@common'
-import { Box, getImageUrl, Icon, MultiSessionBar } from '@components'
-import { useEnvironment, useIsMobileDevice } from '@lib'
-import { getStudyDuration, getStudyPoints, isMultiSession } from '@models'
+import { Box, getImageUrl } from '@components'
+import { useEnvironment, useIsMobileDevice, useApi } from '@lib'
+import { getStudyDuration, getStudyPoints, isMultiSession, getStudyPi, getStudyLead, launchStudy, isStudyLaunchable } from '@models'
 import { ParticipantStudy, Study } from '@api'
 import styled from '@emotion/styled'
 import { colors, media } from '@theme'
 import { StudyDetailsPreview } from './details';
-import dayjs from 'dayjs';
-import { Button, Space } from '@mantine/core';
+import { Button, Flex, Text, HoverCard, Title, Group, Stack, Overlay } from '@mantine/core';
 import { useNavigate } from 'react-router-dom';
+import { IconBoxMultiple, IconCheck, IconHeartFilled, IconInfoCircleFilled, IconBaselineDensityMedium, IconUserFilled } from '@tabler/icons-react'
 
 interface StudyCardProps {
     study: ParticipantStudy
 }
 
-const Card = styled(Box)({
+const Card = styled(Box)<{ studycompleted: boolean, multisession: boolean }>(({ studycompleted, multisession }) => ({
```

Comment:
We want to move away from styled components - i've been slowly trying to rewrite anything using a styledcomponent to just be it's own component. Perhaps this is a good time to rewrite this as a component and use some simple css. let me know if you think if thats too much work or not for this ticket - we can always do it later too

Reply:
This ticket was so huge. Can I do it in the next ticket?